packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -428,7 +428,7 @@ export const ImageInput = ({
           onChange={handleInputChange}
           onClick={handleClick}
           disabled={disabled || isLoading}
-          aria-invalid={invalid}
+          aria-invalid={invalid && 'true'}
```

Comment:
By setting `invalid && 'true'` we ensure that the attribute is either 'true' or undefined (therefore defaults to 'false').

This prevents issues with e.g. a JavaScript implementation passing a truthy value such as 'This is a required field' to the 'invalid' attribute, resulting in invalid markup and bad accessibility.